crates/context_aware_config/src/api/context/handlers.rs

```diff
@@ -399,7 +401,7 @@ fn r#move(
 ) -> superposition::Result<PutResp> {
     use contexts::dsl;
     let req = req.into_inner();
-    let ctx_condition = Value::Object(req.context);
+    let ctx_condition = json!(req.context.into_inner());
```

Comment:
can we let it be Value::Object itself
```suggestion
    let ctx_condition = Value::Object(req.context.into_inner());
```